### ISSUE TITLE
Tag a new version

### ISSUE THREAD

**Issue by dgikiller** [amonakov/primus#169]
Hi,
First of all thank you for your work, I like very much this project.
I'm here to ask if could be possible to tag a new version of primus, because the last tag is really old and last commits works well i think.
Best Regards.

**Comment by akien-mga** [comment]
+1 :)

**Comment by mudler** [comment]
:+1:

**Comment by WhiteWolf1776** [comment]
I actually foked primus just so I could do version bumps after testing the updates for my slackware users.

**Comment by ArchangeGabriel** [comment]
On arch and ubuntu we use yyyymmdd-n release version number. This is easy to use. You can also use rev number, and in both case, at git short commit number to specify the release.

Normally, we release after each serie of change, since @amonakov works this way.

**Comment by dgikiller** [comment]
@ArchangeGabriel I know that is a common practice, but I think that tagging version make more maintainable the software repositories. Moreover I'm asking because the project already have a tag (0.1) but it is 2 years old.

**Comment by ArchangeGabriel** [comment]
Well, once @amonakov will have shown signs of life, maybe he will consider merging pending patches and potentially tag a 1.0 or 1.1 version (because of Fedora version numbers) if needed.

**Comment by zx2c4** [comment]
0.2 was just tagged!

**Comment by amonakov** [comment]
Yep, in a conversation @zx2c4 convinced me to just tag current tree with v0.2 since he wanted to put primus ebuild upstream in Gentoo (which he has just done). Sorry about not taking care of the pending pull request, but I'll try to merge that before the next tag ;)

**Comment by melroy89** [comment]
Version is tagged (0.2): see #190. issue can be closed OK.
